src/js/TuningSliders.js

```diff
@@ -133,10 +135,11 @@ TuningSliders.initPidSlidersPosition = function() {
 };
 
 TuningSliders.initGyroFilterSliderPosition = function() {
-    if (semver.lte(FC.CONFIG.apiVersion, API_VERSION_1_43)) {
+    if (semver.lt(FC.CONFIG.apiVersion, API_VERSION_1_44)) {
```

Comment:
What is the reason for these changes? Pretty much all existing checks in this file are `semver.lte(FC.CONFIG.apiVersion, API_VERSION_1_43)`, so changing this here just adds inconsistency and makes this harder to maintain.

Reply:
No problem, I wanted to use `API_VERSION_1_44` for consistency.

Reply:
I changed everything in the file to be .le for 1_44.

Reply:
To me this is much more readable. @mikeller ?

Reply:
To resolve we could
```
Add line 31: `firmwareSliders: false,`
and line 48: `this.firmwareSliders = semver.gte(FC.CONFIG.apiVersion, API_VERSION_1_44)
```
And use this variable instead of the comparision

Reply:
@haslinghuis , I think we're good.  I just changed all to .lt(44) so they are consistent within the files.  I think 3 were still the other way.  Easy peasy lemon squeezy.